static/docs/user-guide/dvcignore.md

```diff
@@ -38,7 +38,107 @@ by design works similar way as `.gitignore` does.
 
 The same as for [`.gitignore`](https://git-scm.com/docs/gitignore).
 
-## Example
+## Examples: Modification of ignored data
+
+Lets see if what happens when we modify ignored file.
+
+```dvc
+$ mkdir data
+$ echo data1 >> data/data1
+$ echo data2 >> data/data2
+$ tree .
+
+.
+└── data
+    ├── data1
+    └── data2
+```
+
+We created data directory. Lets ignore part of data and add `data` under DVC
```

Comment:
use `data` everywhere as a code block. I also think you need to use `the` - created _the_ data directory. @jorgeorpinel can confirm that :)